tests/func/test_run_multistage.py

```diff
@@ -315,3 +318,43 @@ def test_run_without_cmd(kwargs):
     with pytest.raises(InvalidArgumentError) as exc:
         Repo().run(**kwargs)
     assert "command is not specified" == str(exc.value)
+
+
+def test_run_overwrite_order(tmp_dir, dvc, run_copy):
+    tmp_dir.gen({"foo": "foo", "foo1": "foo1"})
+    run_copy("foo", "bar", name="copy-foo-bar")
+    run_copy("bar", "foobar", name="copy-bar-foobar")
+
+    run_copy("foo1", "bar1", name="copy-foo-bar", force=True)
```

Comment:
don't know if the `force` being default was intentional on Python API, maybe for ease of use?
Anyway, a lot of tests depend of `force=True`.

Reply:
@skshetry Just noticing this myself as well and I think it is a bug, actually.